app/main.py

```diff
@@ -1,8 +1,54 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def can_wash_car(self, car):
+        return self.clean_power >= car.clean_mark
+
+    def calculate_washing_price(self, car):
+        if self.can_wash_car(car):
+            return round(car.comfort_class
+                         * (self.clean_power - car.clean_mark)
+                         * self.average_rating
+                         / self.distance_from_city_center, 1)
+
+    def serve_cars(self, list_of_cars: list):
+        income = 0
+        for car in list_of_cars:
+            if self.can_wash_car(car):
+                income += (car.comfort_class
+                           * (self.clean_power - car.clean_mark)
+                           * self.average_rating
+                           / self.distance_from_city_center)
```

Comment:
don't repeat yourself: use method you wrote

Reply:
But, if i don't mistake, it will be the best practice soon, or no ?)
https://prnt.sc/0nwHZjOTU9Bx

Reply:
aaaa, ok

Reply:
Ok, i will try)

Reply:
Yes, you are right, unfortunately  i didn't think about that i can not to round the result of  serve_cars